Exercises/11.LoopsAndConditions/TheBiggerNum.py

```diff
@@ -8,3 +11,8 @@ def the_bigger_number(list_of_numbers):
         except ValueError:
             raise ValueError(f"{num} is not a number")
     return big
+
+# As a side not - python has a build-in function called max()
```

Comment:
Ok, 
I assumed that the list input may be :
list = ["1", "2", 4.0, "3"]
i was need to use with ? 
print(max(map(int, list)))

Reply:
yeah first convert everything to numeric then call max. I would use list comprehension here instead of map